src/main/resources/queries/tpch/control/sf1000/q9.sql

```diff
@@ -0,0 +1,180 @@
+SELECT *
+FROM
+(
+  VALUES
+    (VARCHAR 'ALGERIA', BIGINT '1998', DECIMAL '27405531574.3185'),
```

Comment:
`DECIMAL '...'` redundant, `27405531574.3185` would produce value of same type & value